cvat/apps/webhooks/serializers.py

```diff
@@ -145,3 +149,6 @@ class Meta:
             "response",
         )
         read_only_fields = fields
+        extra_kwargs = {
```

Comment:
I see `status_code = models.PositiveIntegerField(null=True, default=None)` definition in the models.py file. Why do we need it here as well?

Reply:
Hmm, seems like a mistake. Fixed.